golos.vesting/golos.vesting.cpp

```diff
@@ -162,6 +162,7 @@ void vesting::delegate_vesting(account_name sender, account_name recipient, asse
     eosio_assert(sender != recipient, "You can not delegate to yourself");
 
     eosio_assert(MIN_AMOUNT_DELEGATION_VESTING <= quantity.amount, "Insufficient funds for delegation");
+    eosio_assert(MAX_PERSENT_DELEGATION < percentage_deductions, "Exceeded the percentage of delegated vesting");
```

Comment:
Invalid condition. You check that the percentage set by the user is **greater** than the maximum in the system.